modules/report/view/reskontrovoucherprint.php

```diff
@@ -57,6 +58,13 @@
     $whereouter .= " v.DepartmentID = " . $_REQUEST['report_DepartmentID'] . " and ";
 }
 
+if ( in_array($selectedAccount->AccountPlanID, array(1500, 2400, 2930)))
+{
+  if ($selectedAccount->AccountPlanID == 1500) $whereouter .= " a.AccountPlanType = 'customer' and ";
+  if ($selectedAccount->AccountPlanID == 2400) $whereouter .= " a.AccountPlanType = 'supplier' and ";
+  if ($selectedAccount->AccountPlanID == 2930) $whereouter .= " a.AccountPlanType = 'employee' and ";
+}
+
```

Comment:
Instead of these 3 ifs we should use `$selectedAccount->ReskontroAccountPlanType`